main.cpp

```diff
@@ -430,7 +494,32 @@ Thing 8) Manager
     2) manages the studio
     3) gets a job
  */
-
+struct manager
+{
+  int age;
+  double height;
+  std::string name;
+  std::string gender;
+  double startToWrok;
+
+  struct managerAbility
+  {
+    double degrees = 3.67;
+    bool recordingknowHow = true;
+    std::string school = "Bahcesehir University";
+    std::string schoolDepartment = "Communication";
+    int dateOfGraduation = 2017;
+
+    double wakeUp = 07.10;
+    double enterTheAlarmPin = 3321;
+    double payTheMagazines = 13.2;
+  };
+  void keepTrackOfExpenses(double electricityBill = 322.3, double waterBill = 63.2);
+  void openTheDoors(bool mixingRoomDoor = true, bool recordingRoomDoor = false);
+  void callTheCustomer(std::string customerName = "Haktan", int customerNumber = 534855775);
+
+  managerAbility talentDemonstrated;
+};
```

Comment:
Fix the following:
- Type Name capitalization
- Indentation relative to the `{ }`
- your nested class has no member functions.  It needs 3. 
- [ ] revise